Userland/Libraries/LibCore/LocalSocket.cpp

```diff
@@ -74,6 +79,10 @@ pid_t LocalSocket::peer_pid() const
     if (getsockopt(fd(), SOL_LOCAL, LOCAL_PEERPID, &pid, &pid_size) < 0) {
 #elif defined(__FreeBSD__)
     if (getsockopt(fd(), SOL_LOCAL, LOCAL_PEERCRED, &creds, &creds_size) < 0) {
+#elif defined(__OpenBSD__)
+    int error;
+    error = getsockopt(fd(), SOL_SOCKET, SO_PEERCRED, &creds, &creds_size);
```

Comment:
Isn't this identical to the case we already have in the in the `#else` block below?